src/api/tenantApi.js

```diff
@@ -0,0 +1,116 @@
+import axios from 'axios'
+const BASE_URL = 'https://dev.elevate-apis.shikshalokam.org'
```

Comment:
Base url should be in a .env file  file so that we can just modify it for every environemnts like dev, qa and prod. MAke sure the .env is not part of the PR by adding it to .gitignore. You can put a .env_sample file insted which will  have the key and value could be some sample  values rather than original value